dumper/dataWriter.go

```diff
@@ -219,12 +220,9 @@ func GetRowsFromKeys(db *sql.DB, table schemareader.Table, keys []TableKey) [][]
 }
 
 func filterRowData(value []sqlUtil.RowDataStructure, table schemareader.Table) []sqlUtil.RowDataStructure {
-	if strings.Compare(table.Name, "rhnerrata") == 0 {
-		for i, row := range value {
-			if strings.Compare(row.ColumnName, "severity_id") == 0 {
-				value[i].Value = value[i].GetInitialValue()
-			}
-		}
+	if table.RowModCallback != nil {
+		// value passed as slice so modifications done in callback are visible in this scope as well
+		table.RowModCallback(value, table)
```

Comment:
don't you need to save the returned values to value again? I think go passes the fields as pass by copy, so the out value variable will not be affected

Reply:
I was wondering that too, I initially had it assigned back to `value`, but IIUC my googling we are passing `value` as slice and those are passed as reference. See https://stackoverflow.com/a/40514753

In my tests it indeed exported modified pillar.

Reply:
In that case is fine looks like it will work :)
BUt I would prefer that go kept the coherence and we would need to assign it back :)

Reply:
Re-added assignment back to `value` so it is clear we are doing modifications over the value.